server/src/main/java/org/opensearch/action/search/SearchExecutionStatsCollector.java

```diff
@@ -66,7 +67,24 @@
 
     @Override
     public void onResponse(SearchPhaseResult response) {
+        if (response instanceof QueryFetchSearchResult) {
```

Comment:
Why is this condition necessary? I think the following code will always overwrite network outbound time, and in the local case will overwrite inbound network time too.